src/main/java/no/nav/bidrag/dokument/arkiv/controller/JournalpostController.java

```diff
@@ -131,7 +131,7 @@ private boolean erIkkePrefixetMedJoark(String joarkJournalpostId) {
     }
 
     var endreJournalpostHttpResponse = journalpostService.endre(
-        kildesystemIdenfikator.hentJournalpostId(), new EndreJournalpostCommandIntern(endreJournalpostCommand, enhet)
+        Long.valueOf(kildesystemIdenfikator.hentJournalpostId()), new EndreJournalpostCommandIntern(endreJournalpostCommand, enhet)
```

Comment:
denne er long I joark? opprett et issue for å hente id som long i bidrag-commons og vis til denne kodelinje... jeg ønsker ikke å ha unødvendig wrapping i koden...